README.md

```diff
@@ -21,6 +21,28 @@ A plugin that displays user-friendly messages when Video.js encounters an error.
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Getting Started
+**Importing via NPM/Babel/Browserify**
+```shell
+npm install videojs-errors
+```
+
+You may need to add videojs-errors as plugin to the videojs module:
+```js
+import videojs from 'video.js';
+import videojsErrors from 'videojs-errors';
+
+//For VideoJS 5 and below
+videojs.plugin('errors', videojsErrors);
+
+//For VideoJS 6
+videojs.registerPlugin('errors', videojsErrors);
```

Comment:
Sorry if I was unclear, but what I was saying is that you should never need to do this - [videojs-errors does it already](https://github.com/brightcove/videojs-errors/blob/master/src/plugin.js#L323).

Reply:
see #116 
this is the problem here

Reply:
thanks for reminding us @axten, it'll be fixed in the next release.

Reply:
so, only the following should be needed:
```
import videojs from "video.js";
import "videojs-errors";
```